guides/tutorials/codelabs/gce-source-to-prod/index.md

```diff
@@ -309,8 +299,9 @@ We deploy the image that was picked, to the prod cluster.
 * Select “Deploy” in the *Type* drop down.
 * In the *Server group* section, click *Add server group*.
 * In the dialog that appears, click *Continue without a template*.
-* In the Basic Settings section, enter “prod” in the *Stack* field.
+* In the Basic Settings section, enter "prod" in the *Stack* field.
```

Comment:
I prefer it unordered, but could be convinced if somebody feels strongly about it.